exercises/all-your-base/all-your-base.sml

```diff
@@ -0,0 +1,2 @@
+fun rebase (fromBase: int) (xs: int list) (toBase: int): int list option =
```

Comment:
Any reason why you are using a curried function instead of a tupled one? My reasoning not to generate a curried version is because it won't be used to compose other functions and reading the tests would be easier in the eyes.